collections/_article/parallax-progress-report.md

```diff
@@ -0,0 +1,112 @@
+
+<video autoplay loop muted playsinline title="An example of 2D fake light effect">
+  <source src="/storage/blog/parallax2d/lights.mp4" type="video/mp4">
+</video>
+
+To achieve this lighting effect, my system uses `GradientTexture2D`'s to mask an overlay or foreground objects using a screen-reading shader. For this to work, both the mask and the target need to be in the same `CanvasLayer`. This is normally the case for nodes like `Sprite2D` or `TextureRect`, but `ParallaxLayer` is restricted to being a direct child of `ParallaxBackground`. Not only this (even worse), `ParallaxBackground`'s base class is `CanvasLayer`, making it completely incompatible with this technique. GROSS.
+
+To understand why this is the case, we need to point out that `ParallaxLayer` not only provides a parallax effect, but also an incredibly popular "infinite scrolling" bonus effect. Of course, actual infinite scrolling isn't possible (right?), so, behind the scenes, Godot performs a bit of visual trickery (dramatization):
+
+<video autoplay loop muted playsinline title="Godot mascot riding in a convertible on a stage">
+  <source src="/storage/blog/parallax2d/secret.mp4" type="video/mp4">
+</video>
+
+The textures are repeated by a value you set and their position zips back when it scrolls too far. This makes it appear like it's repeating forever, deceiving the viewer! Godot is practically an illusionist!
+
+Originally, `ParallaxBackground` was intentionally made a `CanvasLayer` (stay with me here) to take advantage of a quirk of the rendering system: each layer is processed one at a time. This provides space to repeat entire branches multiple times, achieving `ParallaxLayer`'s "infinite scrolling" effect. This isn't ideal, though. We want to allow this *anywhere* in the tree, not just at the root of a layer. By passing a field unique to `ParallaxLayer` further down in the renderer, we can have the same effect on a sub-branch. This also reveals something else significant: without the need for `ParallaxBackground` to be a `CanvasLayer`, there's not much of a need for `ParallaxBackground` at all! Aw man! This is starting to sound like a breaking change...
+
+After consulting with a few other contributors, it was clear there were two options:
+1. Wait until 5.0 and make these changes directly to `ParallaxLayer` and retire `ParallaxBackground`.
+2. Make a new node and provide a conversion tool.
+
+BIG SURPRISE: we decided to go with the second option! Having duplicate features could be confusing to users, but because 4.0 only came out a year ago, users may be waiting a long time for these fixes if we push it out to a major release. Though... an entirely new node means we can make *even more* changes!
```

Comment:
> BIG SURPRISE: we decided to go with the second option!

I really like your writing style, and don't mind that it leans more informal and humorous, if anything that makes it a pleasure to read in my book! However, here I would avoid going all caps since that is a little misleading in terms of importancy. I would remove this sentence and simply start the next one with "While..."

Additionally, I would rephrase the last one to "By creating an entirely new node we have the freedom to make these changes immediately, and *even more* of them"